app/models/miq_queue.rb

```diff
@@ -364,6 +364,7 @@ def deliver(requester = nil)
       _log.error("#{MiqQueue.format_short_log_msg(self)}, #{message}")
       status = STATUS_EXPIRED
     rescue Exception => error
+      raise if error.class.name == "IRB::Abort"
```

Comment:
Would it make sense just to add this error to the line a few lines up:

```diff
- rescue SystemExit
+ rescue SystemExit, IRB::Abort
```

We might need to do something like this:

```ruby
def self.abort_exceptions
  @abort_exceptions ||= begin
    exceptions  = [SystemExit]
    exceptions << IRB::Abort if defined?(IRB::Abort)
  end
end
```

And then it would look something like this:

```diff
- rescue SystemExit
+ rescue *self.class.abort_exceptions
```